schemas/post-artifact-request.yml

```diff
@@ -31,11 +31,35 @@ oneOf:
            always using `application/json` for JSON artifacts.
         type:       string
         maxLength:  255
+      contentMD5:
+        description: |
+          Base64 encoded MD5 sum of the artifact. If specified, it **must**
+          also be given as value to the `Content-MD5` header when uploading
+          to S3.
+          **Warning**, this will likely be required in the future, and **must**
+          be specified if `contentSHA256` is specified.
+        type:       string
```

Comment:
What is their "ascii form" are you talking `hex`?

This an encoding of the binary format in base64. For `contentMD5` s3 expects `content-md5` header to be base64 encoded md5 sum. I figured doing the same with sha256 made sense...
It doesn't really matter, base64 is a bit shorter that's all... But we could make the sha256 sum as `hex`.